src/applications/static-pages/scroll-top-button.js

```diff
@@ -6,6 +6,11 @@
  */
 
 function navigateToTop() {
+  const content = document.getElementById('content');
+  if (content) {
+    content.setAttribute('tabindex', '-1');
+    content.focus();
+  }
```

Comment:
Optionally there is a platform function that we could use here

```js
import { focusElement } from 'platform/utilities/ui';

focusElement('#content')
```

Reply:
thanks @ncksllvn i'm gonna try using that since it handles setting the `tabIndex` for you:
```export function focusElement(selectorOrElement, options) {
  const el =
    typeof selectorOrElement === 'string'
      ? document.querySelector(selectorOrElement)
      : selectorOrElement;

  if (el) {
    if (el.tabIndex === 0) {
      el.setAttribute('tabindex', '0');
    }
    if (el.tabIndex < 0) {
      el.setAttribute('tabindex', '-1');
    }
    el.focus(options);
  }
}```